WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -148,6 +155,16 @@ extension ReaderTabView {
         self.filteredTabs.removeAll(where: { $0.index == selectedIndex })
         self.filteredTabs.append((index: selectedIndex, topic: selectedTopic))
     }
+
+    private func updateMenuDisplay(hidden: Bool) {
+        guard isMenuHidden != hidden else { return }
+
+        isMenuHidden = hidden
+        mainStackViewTopAnchor?.constant = hidden ? -buttonContainer.frame.height : 0
+        UIView.animate(withDuration: 0.2) {
```

Comment:
Nit: what do you think about using the constant value for the navigation bar transition duration?

```suggestion
        UIView.animate(withDuration: UINavigationController.hideShowBarDuration) {
```

Reply:
I extracted it to a constant value in https://github.com/wordpress-mobile/WordPress-iOS/pull/22353/commits/9792688c7c65741f1fc0c1402e5bdaa303c3eaec. Since it isn't technically using a navigation controller, I decided to extract it to the existing `Appearance` constants instead.